pkg/jobmanager/status.go

```diff
@@ -45,6 +37,32 @@ func (jm *JobManager) status(ev *api.Event) *api.EventResponse {
 		return &evResp
 	}
 
+	// Is it for our instance?
+	if jm.config.instanceTag != "" {
+		found := false
+		for _, tag := range currentJob.Tags {
+			if tag == jm.config.instanceTag {
+				found = true
+				break
+			}
+		}
+		if !found {
+			evResp.Err = fmt.Errorf("job %d belongs to a different instance, this is %q",
```

Comment:
This is a behaviour change, up until now any instance will happily send a job report. But I agree this is a good change, instances might not even run the exact same version or be unable to resolve some plugins